### PR TITLE
Publish 8.16.2021: Update Windows Team text and background behavior

### PR DESCRIPTION
**Pull request by DanPandre**

### CODE REVIEW

**Comment by PRMerger17** [comment]
@DanPandre : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by DanPandre** [comment]
This is concurrent with https://github.com/MicrosoftDocs/Devices-docs-pr/pull/640 & https://github.com/MicrosoftDocs/windows-docs-pr/pull/5456

Intention is for all of them to go live together, if possible.

**Comment by PRMerger12** [comment]
@MandiOhlinger : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by MandiOhlinger** [comment]
#hold-off : Publish on 8/16/2021

File: memdocs/intune/configuration/device-restrictions-windows-10-teams.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ ms.collection: M365-identity-device-management
 > [!NOTE]
 > [!INCLUDE [not-all-settings-are-documented](../includes/not-all-settings-are-documented.md)]
 
-This article describes some of the Microsoft Intune device restrictions settings that you can configure for Surface Hub devices running [Windows 10 Team](/surface-hub/differences-between-surface-hub-and-windows-10-enterprise).
+This article describes some of the Microsoft Intune device restrictions settings that you can configure for devices running Windows 10 Team, including [Surface Hub devices](/surface-hub/differences-between-surface-hub-and-windows-10-enterprise).
```
Comment (DanPandre) [inline memdocs/intune/configuration/device-restrictions-windows-10-teams.md:36]: This is incorrect. Only Surface Hubs can run Windows 10 Team, but not all Surface Hubs are running Windows 10 Team (some have been migrated to Windows 10 Pro/Enterprise).

So "for Surface Hub devices running Windows 10 Team" is the right way to put it.

**Comment by PRMerger6** [comment]
@DanPandre : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/configuration/device-restrictions-windows-10-teams.md (hunk 1 above)
Comment (DanPandre) [reply memdocs/intune/configuration/device-restrictions-windows-10-teams.md]: Submitted https://github.com/MicrosoftDocs/memdocs/pull/1820/commits/75f3d940669fd417ba97e8085a1fba2a871965ea to address this.

**Comment by MandiOhlinger** [comment]
#sign-off

**Review by MandiOhlinger — APPROVED** [review]

**Pull request merged**